<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="ver">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;PrograShop&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="Espectador">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Modo Espectador&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="dash_boton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
          </property>
          <property name="text">
           <string>Dashboard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="download">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
          </property>
          <property name="text">
           <string>Descargar Imagen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_button">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
          </property>
          <property name="text">
           <string>salir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="recortar">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
            </property>
            <property name="text">
             <string>Recortar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="balde">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
            </property>
            <property name="text">
             <string>Balde</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="blurry">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
            </property>
            <property name="text">
             <string>Blurry</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="foto">
          <property name="styleSheet">
           <string notr="true">QLabel{

background-color: rgb(213, 213, 213)

}</string>
          </property>
          <property name="text">
           <string>Lo cambiable</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignBottom">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Comentarios, Yeah!</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QTextBrowser" name="CajaComentarios">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>400</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLineEdit" name="hace_comentarios">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QPushButton" name="atrevete">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background: rgb(0, 0, 0);
border-radius: 10px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;



}</string>
              </property>
              <property name="text">
               <string>Enviar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
